packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -11,42 +43,28 @@ export interface DrawerPanelContentProps extends React.HTMLProps<HTMLDivElement>
   /** Flag indicating that the drawer panel should not have a border. */
   hasNoBorder?: boolean;
   /** Default width for drawer panel */
-  width?: 25 | 33 | 50 | 66 | 75 | 100;
-  /** Drawer panel width on large viewports */
-  widthOnLg?: 25 | 33 | 50 | 66 | 75 | 100;
-  /** Drawer panel width on xl viewports */
-  widthOnXl?: 25 | 33 | 50 | 66 | 75 | 100;
-  /** Drawer panel width on 2xl viewports */
-  widthOn2Xl?: 25 | 33 | 50 | 66 | 75 | 100;
+  breakpointMods?: DrawerBreakpointMod[];
 }
 
 export const DrawerPanelContent: React.SFC<DrawerPanelContentProps> = ({
   className = '',
   children,
   hasNoBorder = false,
-  width,
-  widthOnLg,
-  widthOnXl,
-  widthOn2Xl,
+  breakpointMods = [] as DrawerBreakpointMod[],
   ...props
-}: DrawerPanelContentProps) => (
-  <DrawerContext.Consumer>
-    {({ isExpanded, isStatic }) => (
-      <div
-        className={css(
-          styles.drawerPanel,
-          hasNoBorder && styles.modifiers.noBorder,
-          width && styles.modifiers[`width_${width}` as keyof typeof styles.modifiers],
-          widthOnLg && styles.modifiers[`width_${widthOnLg}OnLg` as keyof typeof styles.modifiers],
-          widthOnXl && styles.modifiers[`width_${widthOnXl}OnXl` as keyof typeof styles.modifiers],
-          widthOn2Xl && styles.modifiers[`width_${widthOn2Xl}On_2xl` as keyof typeof styles.modifiers],
-          className
-        )}
-        hidden={isStatic ? false : !isExpanded}
-        {...props}
-      >
-        {children}
-      </div>
-    )}
-  </DrawerContext.Consumer>
-);
+}: DrawerPanelContentProps) => {
+  const t = formatBreakpointMods(breakpointMods, styles);
+  return (
+    <DrawerContext.Consumer>
+      {({ isExpanded, isStatic }) => (
+        <div
+          className={css(styles.drawerPanel, hasNoBorder && styles.modifiers.noBorder, t, className)}
```

Comment:
Sorry to be a style nit, but can you just replace `t` with `formatBreakpointMods(breakpointMods, styles);`?